strings/tips.txt

```diff
@@ -1,7 +1,6 @@
-Where the space map levels connect is randomized every round, but are otherwise kept consistent within rounds. Remember that they are not necessarily bidirectional!
 You can catch thrown items by toggling on your throw mode with an empty hand active.
-To crack the safe in the vault, use a stethoscope or three plastic explosives on it.
-You can climb onto a table by dragging yourself onto one. This takes time and drops the items in your hands on the table. Clicking on a table that someone else is climbing onto will knock them down.
+To crack a safe, use a stethoscope or three plastic explosives on it.
+You can climb onto a table by dragging yourself onto one. This takes a some time. Clicking on a table that someone else is climbing onto will knock them down.
```

Comment:
Slight mistake with "a some time" instead of "some time" unless I can't grammar

Reply:
Oh oops I’ll fix that A